### PR TITLE
Minor changes for nectar upgrade to latest comit

### PR DESCRIPTION
**Pull request by rishflab**
needed for https://github.com/coblox/nectar/pull/112

### CODE REVIEW

File: comit/src/asset/ethereum/erc20.rs [hunk 1]
```diff
@@ -100,6 +100,12 @@ impl From<Erc20Quantity> for blockchain_contracts::ethereum::TokenQuantity {
     }
 }
 
+impl From<BigUint> for Erc20Quantity {
```
Comment (thomaseizinger) [inline comit/src/asset/ethereum/erc20.rs:103]: NACK. There is a `TryFrom` conversion that you can use which maintains the MAX_VALUE invariant.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]

File: comit/src/expiries.rs [hunk 2]
```diff
@@ -326,9 +326,9 @@ impl<A, B> fmt::Display for Expiries<A, B> {
 
 // Its super easy to mix these up, add types so the compiler saves us.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
-pub struct AlphaOffset(Duration);
+pub struct AlphaOffset(pub Duration);
```
Comment (thomaseizinger) [inline comit/src/expiries.rs:329]: Hmm, not sure if these should be public. What do you think @tcharding?

There exist `From` conversions, so one could argue that if I can convert any `Duration` into one of these, then they might as well be public.

**Review by thomaseizinger — APPROVED** [review]

File: comit/src/expiries.rs (hunk 2 above)
Comment (D4nte) [reply comit/src/expiries.rs:329]: As a general rule I'd say that if you need conversion, then use the `From`/`Into` traits.
Conversion is not a good enough excuse to expose internals.

Comment (thomaseizinger) [reply comit/src/expiries.rs:329]: @rishflab Can you use the conversion functions please instead of making these public?

Comment (rishflab) [reply comit/src/expiries.rs]: doen

**Pull request merged**